tempto-kafka/src/main/java/io/prestodb/tempto/fulfillment/table/kafka/KafkaMessageContentsBuilder.java

```diff
@@ -39,44 +40,38 @@ public KafkaMessageContentsBuilder appendBytes(byte[] bytes) {
         return this;
     }
 
-    public KafkaMessageContentsBuilder appendBytes(int ... bytes) {
+    public KafkaMessageContentsBuilder appendBytes(int... bytes)
+    {
         for (int b : bytes) {
             checkArgument(b >= 0 && b <= 255, "Byte must be in 0-255 range");
             contents.write(b);
         }
         return this;
     }
 
-    public KafkaMessageContentsBuilder appendIntLittleEndian(int v) {
-        contents.write((v & 0xff000000) >> 24);
-        contents.write((v & 0xff0000) >> 16);
-        contents.write((v & 0xff00) >> 8);
-        contents.write(v & 0xff);
+    public KafkaMessageContentsBuilder appendIntBigEndian(int v)
+    {
+        for (int i = 3; i >= 0; --i) {
+            contents.write((v & 0xff << i * 8) >> (i + 1) * 8);
```

Comment:
Is this correct? For `i=0` you are going to `(v & 0xff)` (low 8 bits) and then `>> 8`, i.e. `0`...?
what about one shift less `v >> (i * 8)) & 0xff` ?

Reply:
Yeah - right - not sure while I made +1

Reply:
`v >> (i * 8)) & 0xff` seems fine

Reply:
And also my code is broken as I do not have parens around `(i*8)`